stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.scss

```diff
@@ -27,40 +28,45 @@
 
     .icon-bar {
       display: none;
-
       @include devicebreak(mediumonlytab) {
-        display: block;
-        margin-left: $spacing-12;
+        background:url(#{$img-path-Iconfourback}/Bars.svg) no-repeat 0 0/1.375rem 1.375rem;
       }
+      @extend %header-icon;
     }
 
     .icon-close {
       display: none;
-      margin-left: $spacing-12;
+      @include devicebreak(mediumonlytab) {
+        background:url(#{$img-path-Iconfourback}/Times-blue.svg) no-repeat -0.4375rem -0.3125rem/2.125rem 2.125rem;
+      }
+      @extend %header-icon;
```

Comment:
@Ashish-srij CountrySiteHeader.scss and GlobalHeader.scss have same code for 'icon-bar' and 'icon-close' classes so can we create placeholder for these code

Reply:
@surbhi90 done.